internal/cephfs/volume.go

```diff
@@ -106,14 +106,20 @@ func (vo *volumeOptions) getSubVolumeInfo(ctx context.Context, volID volumeID) (
 
 	bc, ok := info.BytesQuota.(fsAdmin.ByteCount)
 	if !ok {
-		// info.BytesQuota == Infinite
-		return nil, fmt.Errorf("subvolume %s has unsupported quota: %v", string(volID), info.BytesQuota)
+		// we ignore info.BytesQuota == Infinite and just continue
+		// without returning quota information
+		if info.BytesQuota == fsAdmin.Infinite {
```

Comment:
the quota will be set to infinite for the kernel <4.17 which doesn't support quotas.

During Snapshot creation operation we get the subvolume info and return the subvolume size as the snapshot size. if we return  `0` when the user tries to create 1Gb PVC, the external provisioner will validate the size as requested size is greater it will allow for the restore operation, I think it should be fine.

Reply:
uh, indeed! The `==` is intended to be `!=` here :facepalm: